src/status_im/contexts/settings/advanced/view.cljs

```diff
@@ -0,0 +1,259 @@
+   "WARN"  "WARN"
+   "INFO"  "INFO"
+   "DEBUG" "DEBUG"})
+
+(defn- log-level-sheet
+  []
+  (let [current-log-level (rf/sub [:log-level/current-profile-log-level])]
+    [option-picker-sheet
+     {:title            (i18n/label :t/log-level)
+      :options          (keys log-levels)
+      :option-selected? (fn [log-level-id]
+                          (= (log-levels log-level-id) current-log-level))
+      :option-title-fn  #(-> % log-levels string/capitalize)
+      :on-press         (fn [log-level-id]
+                          (rf/dispatch [:advanced-settings/change-log-level log-level-id])
+                          (rf/dispatch [:hide-bottom-sheet]))}]))
+
+(defn- fleet-sheet
+  []
+  (let [current-fleet (rf/sub [:fleets/current-fleet])]
+    [option-picker-sheet
+     {:title            (i18n/label :t/fleet)
+      :options          (map keyword fleets/fleets)
+      :option-selected? (fn [fleet]
+                          (= fleet current-fleet))
+      :option-title-fn  name
+      :on-press         (fn [fleet]
+                          (rf/dispatch [:advanced-settings/change-fleet fleet]))}]))
+
+(defn- copy-string-callback
+  [content property-copied]
+  (fn []
+    (clipboard/set-string (str content))
+    (rf/dispatch
+     [:toasts/upsert
+      {:id   :string-copied
+       :type :positive
+       :text (str property-copied " copied to clipboard")}])))
+
+(defn- open-waku-settings
+  []
+  (rf/dispatch [:show-bottom-sheet
+                {:content waku-backup-settings
+                 :shell?  true
+                 :blur?   true
+                 :theme   :dark}]))
+
+(defn- toggle-peer-syncing
+  []
+  (rf/dispatch [:profile.settings/toggle-peer-syncing]))
+
+(defn- open-log-level-sheet
+  []
+  (rf/dispatch [:show-bottom-sheet
+                {:content log-level-sheet
+                 :theme   :dark
+                 :shell?  true
+                 :blur?   true}]))
+
+(defn- open-fleet-sheet
+  []
+  (rf/dispatch [:show-bottom-sheet
+                {:content fleet-sheet
+                 :theme   :dark
+                 :shell?  true
+                 :blur?   true}]))
+
+(defn- force-crash
+  []
+  (rf/dispatch [:app-monitoring/intended-panic "status-mobile intentional panic"]))
+
+(defn- toggle-light-client
+  []
+  (rf/dispatch [:advanced-settings/toggle-light-client]))
+
+(defn view
+  []
+  (let [log-level (rf/sub [:log-level/current-profile-log-level])
+        analytics-user-id (rf/sub [:centralized-metrics/user-id])
+        light-client-enabled? (rf/sub [:profile/light-client-enabled?])
+        peers-count (rf/sub [:peer-stats/count])
+        current-mailserver (rf/sub [:mailserver/current-name])
+        peer-syncing-enabled? (rf/sub [:profile/peer-syncing-enabled?])
+        {:keys [backup-enabled?
+                default-sync-period
+                last-backup]} (rf/sub [:profile/profile])
+        current-fleet (rf/sub [:fleets/current-fleet])
+        options
+        (rn/use-memo
+         (fn []
```

Comment:
probably better to move it to defined function

Reply:
I didn't notice `lint-fix` moved it like this, I'll refactor it :+1: